### ISSUE TITLE
#bookMeta element no longer available

### ISSUE THREAD

**Issue by strtdusty** [wzub/goodreads_overdrive#3]
Today is the first time I tried using this extension and it does not seem to work.  Debugging shows that in overdrive_goodreads.js goodreadsPageMeta is always null.  Goodreads must have stopped sending that content.  The stars seem to be available using

`
goodreadsPage.querySelector("#ReviewsSection > div:nth-child(6) > div.ReviewsSectionStatistics > div.ReviewsSectionStatistics__ratingStatistics > div > div:nth-child(1) > div").innerText
`
but I'm not sure how fragile that would be.

**Comment by wzub** [comment]
Thank you for highlighting this! Goodreads has recently launched a redesign of the Book page, so as you've also noticed things have moved around a bit. I was able to fix it, but I can't push out an update on the extensions store until I update to manifest v3, which I will try to do over the next few days.

Edit: Actually scratch that - it's only the Edge store that's not letting me upload a new version with v2 - so I've just uploaded the new version on Chrome and Firefox.
